db-schema/mongo/AuthorDetail.js

```diff
@@ -2,20 +2,22 @@ import { mongoose } from '../../db/mongoose.js'
 
 
 const authorDetailSchema = new mongoose.Schema({
-    authorId: {
+    author_id: {
```

Comment:
overall we use camel casing in sprinto, it would be useful to write a guideline of casing followed across the assignment

Reply:
Noted, I have also updated the convention I am using in the description.